web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -395,15 +396,13 @@ export type ReviewAccessRequestParams = {
   reason: string;
   roles: string[];
   id: string;
+  assumeStartTime?: Timestamp;
 };
 
-export type CreateAccessRequestParams = {
-  rootClusterUri: uri.RootClusterUri;
-  reason: string;
-  roles: string[];
-  suggestedReviewers: string[];
-  resourceIds: { kind: ResourceKind; clusterName: string; id: string }[];
-};
+export type CreateAccessRequestParams =
+  apiService.CreateAccessRequestRequest & {
+    rootClusterUri: uri.RootClusterUri;
```

Comment:
Only because of your comment under that PR with dates I noticed that you want to backport to v14 too. In that case, our advice with using `types.CreateAccessRequestParams` directly and not mapping it goes out of the window, because v14 doesn't have protobuf-ts.

This means on v14 we won't be able to simply do…

```
export type CreateAccessRequestParams =
  apiService.CreateAccessRequestRequest & …
```

…because on that branch, `apiService.CreateAccessRequestRequest` is a class and not a simple object.

IIRC, you can try doing this and see where it takes you:

```
export type CreateAccessRequestParams =
  apiService.CreateAccessRequestRequest.AsObject & …
```